autoJoin.py

```diff
@@ -84,6 +95,16 @@ def check_and_join_meeting():
     if elem.get_attribute('aria-pressed') == 'true':  # turn off microphone
         elem.click()
     wait_and_find_element_by_xpath('//button[.="Join now"]').click()  # join meeting
+    global check
+    global meeting_name
+    meeting_name=browser.title
+    check = 0
+    #sending message to my number 
+    client.messages.create(
+    to='"'+data['your_no']+'"', 
+    from_='"'+data['twillio_no']+'"',
+    body='"'+"Hello " +data['nickname']+" I  joined the meeting named ** "+ meeting_name+ "** at time ** " + format(datetime.now()) + "   Hope you are doing well" + '"')
+    #message recived from that number
```

Comment:
Put this code in an if condition. To be run only if the user wants to use twilio